src/components/ProgressCircular/index.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+import AssistiveText from './../AssistiveText';
+import normalizeValue from './normalizeValue';
+import ProgressRing from './progressRing';
+import './styles.css';
+
+/**
+ * ProgressCircular component communicates to the user the progress of a particular process.
+ */
+export default function ProgressCircular({ value, variant, assistiveText }) {
```

Comment:
all components must have a className and style props
also when there is more than two props passed instead use props as argument and then make destructuring in first line inside the function as:
```
export default function ProgressCircular(props) {
    const { value, variant, assistiveText } = props;
```